hashtagmap-web/front/src/store/index.js

```diff
@@ -8,37 +8,56 @@ export default new Vuex.Store({
   state: {
     kakaoMap: "",
     kakaoMapApi: "",
-    isShowDetailModal: false,
-    nowPlaceByDetailModal: "",
-    places: [],
+    detailModal: {
+      isShow: false,
+      placeName: "",
+      hashtagCount: "",
+      posts: "",
```

Comment:
```suggestion
      posts: [],
```
아까 오프라인 코드리뷰에서 이야기했던 부분 수정 안됐어요~